client/a8c-for-agencies/sections/marketplace/checkout/style.scss

```diff
@@ -230,3 +231,40 @@
 .checkout__client-referral-form {
 	padding-block-start: 48px;
 }
+
+.checkout__aside--client {
+	position: relative;
+
+	.checkout__aside-actions {
+		margin-block-start: 16px;
+	}
+	.checkout__summary-client-payment-notice {
+		margin-block-start: 48px;
+		color: var(--color-neutral-70);
+		font-size: rem(16px);
+		font-weight: 600;
+		text-wrap: nowrap;
+	}
+
+	.checkout__summary-notice {
+		margin-block-start: 48px;
+	}
+
+	.checkout__aside-footer {
+		position: absolute;
```

Comment:
Maybe we should avoid using absolute positioning and instead use flex (space-between) due to scalability issues. wdyt?


Here is a checkout with 10 items, causing the footer to overlap.

<img width="623" alt="Screenshot 2024-06-13 at 10 05 15 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/351fa4e7-13be-4ca0-9171-2e2406cb4b7d">